src/current/v23.1/sso-sql.md

```diff
@@ -40,10 +40,11 @@ You must have the ability to update your cluster settings, which can be achieved
 
 You must configure the [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) in the following table to enable JWT authentication to your cluster. Refer to the [Update your cluster settings](#update-your-cluster-settings) section to configure your cluster settings.
 
-| Cluster Setting | Description 
-|-----------------|------ 
-| `server.jwt_authentication.enabled` | Defaults to `false`, must be set to `true` to enable embedded JWT generation.
-| `server.jwt_authentication.jwks` | A list of public signing keys for allowed IdPs; must include your IdP's key. 
+| Cluster Setting | Description
+|-----------------|------
+| `server.jwt_authentication.enabled` | Defaults to `false`, must be set to `true` to enable embedded JWT generation..
+| `server.jwt_authentication.jwks` | A list of public signing keys for allowed IdPs; must include your IdP's key. If `server.jwt_authentication.jwksAutoFetchEnabled` is `true`, keys are fetched automatically and `server.jwt_authentication.jwks` is ignored.
+| {% include_cached new-in.html version="v23.1" %}`server.jwt_authentication.jwksAutoFetchEnabled` | If `true`, public signing keys are automatically fetched from the domains specified in `server.jwt_authentication.issuers` for each session, and `server.jwt_authentication.jwks` is ignored. Defaults to `false`.
```

Comment:
`server.jwt_authentication.jwks_auto_fetch.enabled` instead of `server.jwt_authentication.jwksAutoFetchEnabled` as Srithar pointed out above.